src/scala/scripts/ScalaPBGenerator.scala

```diff
@@ -39,7 +40,13 @@ class ScalaPBGenerator extends Processor {
       val relativePath = root.relativize(fullPath)
 
       relativePath.toFile.getParentFile.mkdirs
-      Files.copy(fullPath, relativePath)
+      Try(Files.copy(fullPath, relativePath)) match {
+        case Failure(err) if err.isInstanceOf[FileAlreadyExistsException] =>
```

Comment:
nit: `case Failure(err: FileAlreadyExistsException) => ` would be a bit less verbose.